dizoo/mujoco/config/ant_gail_sac_config.py

```diff
@@ -0,0 +1,103 @@
+from easydict import EasyDict
```

Comment:
unify name, `xxx_sac_gail_config.py`

Reply:
Noted

Reply:
I will apply this format in all gan configs in the next commit.